pkgs/applications/misc/cura/plugins.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, python3Packages }:
+
+let
+
+  self = {
+
+    octoprint = stdenv.mkDerivation rec {
+      name = "Cura-OctoPrintPlugin";
+      version = "3.5";
+
+      src = fetchFromGitHub {
+        owner = "fieldOfView";
+        repo = "Cura-OctoPrintPlugin";
+        rev = version;
+        sha256 = "1cjfjp9ds411m4w845c60ddiw1mpawrnc435g8cmnl4llxdmjkw1";
+      };
+
+      nativeBuildInputs = [ cmake ];
+
+      propagatedBuildInputs = with python3Packages; [
+        netifaces
+      ];
+
+      meta = with stdenv.lib; {
+        description = "Enables printing directly to OctoPrint and monitoring the process";
+        homepage = https://github.com/fieldOfView/Cura-OctoPrintPlugin;
```

```suggestion
        homepage = "https://github.com/fieldOfView/Cura-OctoPrintPlugin";
```

URLs should be quoted now: https://github.com/NixOS/rfcs/pull/45